CONTRIBUTING.md

```diff
@@ -0,0 +1,267 @@
+
+## You can contribute in so many ways!
+
+The repository is split into different parts, where we welcome contributions.
+
+### The client library, `huggingface_hub`
+
+This repository hosts the client library `huggingface_hub`, which is a frontend to the Hugging Face Hub.
+this part lives in `src/huggingface_hub` and `tests`.
+
+There are many ways you can contribute to this client library:
+* Fixing outstanding issues with the existing code;
+* Contributing to the examples or to the documentation;
+* Submitting issues related to bugs or desired new features.
+
+When opening a PR on this part of the repository, please add as reviewer:
+- @LysandreJik
+- @osanseviero
+
+Additionally, here are the owners of specific parts of the library:
+- Core of the library: @muellerzr
+- Documentation: @stevhliu
+- Mixins: @nateraw
+
+### The community inference API
+
+The `api-inference-community` folder contains a tool to enable third-party library support integrated with the Hugging
+Face Hub.
+
+We welcome contributions to [add new containers](https://huggingface.co/docs/hub/adding-a-library#set-up-the-inference-api) for new libraries, to update the existing ones, and to provide help
+fixing bugs and adding features.
+
+When opening a PR on this part of the repository, please add @Narsil as a reviewer.
+
+## JavaScript content
+
+In the `js` folder live the JavaScript code of the Hub. It includes:
+* The widgets ([code](https://github.com/huggingface/huggingface_hub/tree/main/js/src/lib/components/InferenceWidget)
+* Code snippets to make inference calls ([code](https://github.com/huggingface/huggingface_hub/tree/main/js/src/lib/inferenceSnippets)
+* Code snippets to load models ([code](https://github.com/huggingface/huggingface_hub/blob/main/js/src/lib/interfaces/Libraries.ts))
+
+Here too, we welcome any logic and documentation contributions.
+
+When opening a PR on this part of the repository, please add @mishig25 and @osanseviero as a reviewer.
+
+*All contributions are equally valuable to the community.*
+
+## Submitting a new issue or feature request
+
+Do your best to follow these guidelines when submitting an issue or a feature
+request. It will make it easier for us to come back to you quickly and with good
+feedback.
+
+### Did you find a bug?
+
+The 🤗 Hugging Face Hub library is robust and reliable thanks to the users who notify us of
+the problems they encounter. So thank you for reporting an issue.
+
+First, we would really appreciate it if you could **make sure the bug was not
+already reported** (use the search bar on Github under Issues).
+
+Did not find it? :( So we can act quickly on it, please follow these steps:
+
+* Include your **OS type and version**, the versions of **Python**, **PyTorch** and
+  **Tensorflow** when applicable;
+* A short, self-contained, code snippet that allows us to reproduce the bug in
+  less than 30s;
+* Provide the *full* traceback if an exception is raised.
```

Comment:
Perhaps change the wording to be include a copy (as text) of the *full* traceback....

So they don't just add screenshots of it (this I've seen more beginners do)